include/vkgcDefs.h

```diff
@@ -793,6 +794,10 @@ struct GraphicsPipelineBuildInfo {
                                   ///  matches the sample pattern used by the rasterizer when rendering
                                   ///  with this pipeline.
     uint8_t usrClipPlaneMask;     ///< Mask to indicate the enabled user defined clip planes
+
+    VkProvokingVertexModeEXT provokingVertexMode; ///< Specifies which vertex of a primitive is the _provoking
```

Comment:
please move usrClipPlaneMask before numSamples  to group all 32bit member together.

Reply:
Fixed, thanks.